subsys/net/l2/canbus/Kconfig

```diff
@@ -0,0 +1,40 @@
+
+config NET_L2_CANBUS
+	bool "Enable CANBUS L2 layer [Experimental]"
+	depends on CAN_NET
+	select NET_6LO
+	help
+	  Add a CANBUS L2 layer driver. This is the layer for IPv6 over CAN
+	  (6loCAN). It uses IPHC to compress the IP header and ISO-TP for
+	  flow control and reassembling.
+
+if NET_L2_CANBUS
+
+config NET_L2_CANBUS_USE_FIXED_ADDR
+	bool "Use fixed L2 address"
+	help
+	  Use a fixed L2 address for 6LoCAN instead of a random chosen one
+
+config NET_L2_CANBUS_FIXED_ADDR
+	int "L2 address"
```

Comment:
Just pondering this but would it make sense to use hex value here, then it is easier to correlate the value if looking through message dumps etc.